### DISCUSSION TITLE
更新1.10.5 LTS之后容器列表内的关联资源出错

### DISCUSSION THREAD

**Discussion by M1saka10010** [1Panel-dev/1Panel#5061] [Q&A]
### 联系方式

admin@misaka.ren

### 1Panel 版本

1.10.5 LTS

### 问题描述

不影响实际的网站运行以及配置文件，一个网站指向到phpmyadmin去了（我停用了一段时间了，另一个网站的配置文件指向到这个指向到pma的配置文件这里了）只是管理界面有问题，我看1panel的数据库也暂时看不出问题。而且nginx的配置文件显示正常，只有php的配置文件显示有误。

### 重现步骤

更新了面板然后更新了nginx就出现这个问题了。

### 期待的正确结果

后台各归原位。

**Comment by zhengkunwang223** [comment]
截图看一下

**Comment by M1saka10010** [comment]
![image](https://github.com/1Panel-dev/1Panel/assets/29598885/7a9519ae-6138-450e-a734-04878595b75f)


> 截图看一下

**Comment by M1saka10010** [comment]
手动修改sqlite后恢复，猜测为重建nginx导致的install app id变更

**Reply by zzzxp** [reply]
请问怎么修改的，我也碰到这个问题

**Comment by zhengkunwang223** [comment]
> 手动修改sqlite后恢复，猜测为重建nginx导致的install app id变更

是不是从其他 1panel 恢复过网站备份

**Comment by M1saka10010** [comment]
> > 手动修改sqlite后恢复，猜测为重建nginx导致的install app id变更
> 
> 是不是从其他 1panel 恢复过网站备份


你这么说应该是了

**Comment by zhengkunwang223** [comment]
> > > 手动修改sqlite后恢复，猜测为重建nginx导致的install app id变更
> 
> > 
> 
> > 是不是从其他 1panel 恢复过网站备份
> 
> 
> 
> 
> 
> 你这么说应该是了

当前的网站备份恢复确实会导致这个问题，后续我们会通过增加网站导入功能来支持从其他 1panel 恢复网站
